misc/de-metas-common/de-metas-common-rest_api/src/main/java/de/metas/common/rest_api/v1/SyncAdvise.java

```diff
@@ -126,10 +139,12 @@ public boolean isFailIfNotExists()
 		return IfNotExists.FAIL.equals(ifNotExists);
 	}
 
-	/** If {@code true} then the sync code can attempt to lookup readonlydata. Maybe this info helps with caching. */
+	/**
+	 * If {@code true} then the sync code can attempt to lookup readonlydata. Maybe this info helps with caching.
+	 */
 	@JsonIgnore
 	public boolean isLoadReadOnly()
 	{
-		return READ_ONLY.equals(this);
+		return READ_ONLY.equals(this) || READ_ONLY_UNCHANGED.equals(this);
```

Comment:
a bit fragile IMHO, but let's keep it as it is for now (if no more changes are required here).

If we need to touch this part, I will introduce the isLoadReadonly() method to both IfExists and IfNotExists enums,
and here it would just be "logical and" them.